File: 7/27/4.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;

    string tmp = to_string(n);

    reverse(tmp.begin(),tmp.end());

    for(int i = 0; i <tmp.size();i ++){
        answer.push_back(tmp[i]-48);

    }

    return answer;
}

File: 2021/2/28/4.py
class Solution:
    def reverse(self, x) -> int:
        s = str(x)
        if s[0] == '-' :
            s = s[1:]
            s = s[::-1]
            ans = int(s)*-1
            if ans < -1*pow(2,31) :
                return 0


            return ans

        else :
            s = s[::-1]
            ans = int(s)
            if ans >= pow(2,31) :
                return 0
            return ans